Sources/Html/Attributes.swift

```diff
@@ -293,6 +293,19 @@ public func draggable<T>(_ value: Draggable) -> Attribute<T> {
   return .init("draggable", value == .auto ? nil : value.rawValue)
 }
 
+public enum Enctype: String {
+  case formURLEncoded = "application/x-www-form-urlencoded"
+  case multipartFormData = "multipart/form-data"
+  case plainText = "text/plain"
+}
+
+/// The type of form encoding.
+///
+/// - Parameter value: Enctype to use for form encoding.
+public func enctype(_ value: Enctype) -> Attribute<Tag.Form> {
```

Comment:
We have a few `element` helpers that promote even more type safety. For example, `bdo` tags require a `dir` attribute:

https://github.com/pointfreeco/swift-html/blob/master/Sources/Html/Elements.swift#L286-L288

Since `enctype` requires that the form have a `method` of `post`, I wonder if we want to also/instead add a `form` helper?

``` swift
func form(enctype: Enctype, _ attribs: [Attribute<Tag.Form>], _ content: [Node]) -> Node {
  return form([method(.post), .init("enctype", enctype.rawValue)] + attribs, content)
}
```

What do ya think? Want to add it to this PR or should I add it after we get it merged?